user-guide/layer-manager.rst

```diff
@@ -0,0 +1,45 @@
+Layer Manager
+=============
+
+The Layer manager can be accessed from the menu by selecting **Tools > Layer Manager**. The layer manager will give the user flexibility to adjust the order in which items are displayed on the map.
+
+.. image:: images/layer-manager-01.png
+
+Ordering layers
+^^^^^^^^^^^^^^^
+
+Layers are prioritized from the top to the bottom. Layers on the top will overlap any layers under them.
+
+To readjust the order of a selected layer, the user will need to use the up and down arrows that are provided on the right. The single up and down arrow is used to move the selected layer one placement in either direction. The double up and down arrows are used to move the selected layer to the top or bottom of all the other layers displayed.
```

Comment:
layer -> layer(s) when talking about selected layer, as multiple layers can be selected. Also, may want to add a note about drag and drop.